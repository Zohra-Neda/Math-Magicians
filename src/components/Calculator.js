import React from 'react';
import Container from './CalcContainer';
import Buttons from './Buttons';

const Calculator = () => (
  <div className="main">
    <Container />
    <Buttons />
  </div>
);

export default Calculator;
