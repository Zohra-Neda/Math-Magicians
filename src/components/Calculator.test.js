import React from 'react';
import { render } from '@testing-library/react';
import Calculator from './Calculator';

test('renders Calculator component correctly', () => {
  const { asFragment } = render(<Calculator />);
  expect(asFragment()).toMatchSnapshot();
});
