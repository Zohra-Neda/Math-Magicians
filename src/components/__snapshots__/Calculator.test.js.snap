// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Calculator component correctly 1`] = `
<DocumentFragment>
  <div
    class="calculator-content"
  >
    <h2
      class="calculator-title"
    >
      Lets do some math!
    </h2>
    <div
      class="calculator"
    >
      <div
        class="calc-container"
      >
        0
      </div>
      <div
        class="buttons"
      >
        <button
          class="btn gray"
          type="button"
        >
          AC
        </button>
        <button
          class="btn gray"
          type="button"
        >
          +/-
        </button>
        <button
          class="btn gray"
          type="button"
        >
          %
        </button>
        <button
          class="btn orange"
          type="button"
        >
          รท
        </button>
        <button
          class="btn gray"
          type="button"
        >
          7
        </button>
        <button
          class="btn gray"
          type="button"
        >
          8
        </button>
        <button
          class="btn gray"
          type="button"
        >
          9
        </button>
        <button
          class="btn orange"
          type="button"
        >
          x
        </button>
        <button
          class="btn gray"
          type="button"
        >
          4
        </button>
        <button
          class="btn gray"
          type="button"
        >
          5
        </button>
        <button
          class="btn gray"
          type="button"
        >
          6
        </button>
        <button
          class="btn orange"
          type="button"
        >
          -
        </button>
        <button
          class="btn gray"
          type="button"
        >
          1
        </button>
        <button
          class="btn gray"
          type="button"
        >
          2
        </button>
        <button
          class="btn gray"
          type="button"
        >
          3
        </button>
        <button
          class="btn orange"
          type="button"
        >
          +
        </button>
        <button
          class="btn gray span"
          type="button"
        >
          0
        </button>
        <button
          class="btn gray"
          type="button"
        >
          .
        </button>
        <button
          class="btn orange"
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
